MRTK3 Tutorials/Assets/MRTK.Tutorials.AzureSpatialAnchors/Scripts/AnchorModuleScript.cs

```diff
@@ -0,0 +1,487 @@
+
+        // Start watching for Anchors
+        if ((cloudManager != null) && (cloudManager.Session != null))
+        {
+            currentWatcher = cloudManager.Session.CreateWatcher(anchorLocateCriteria);
+            Debug.Log("Watcher created");
+            Debug.Log("Looking for Azure anchor... please wait...");
+        }
+        else
+        {
+            Debug.Log("Attempt to create watcher failed, no session exists");
+            currentWatcher = null;
+        }  
+    }
+
+    public async void DeleteAzureAnchor()
+    {
+        Debug.Log("\nAnchorModuleScript.DeleteAzureAnchor()");
+
+        // Notify AnchorFeedbackScript
+        OnDeleteASAAnchor?.Invoke();
+
+        // Delete the Azure anchor with the ID specified off the server and locally
+        await cloudManager.DeleteAnchorAsync(currentCloudAnchor);
+        currentCloudAnchor = null;
+
+        Debug.Log("Azure anchor deleted successfully");
+    }
+
+    public void SaveAzureAnchorIdToDisk()
+    {
+        Debug.Log("\nAnchorModuleScript.SaveAzureAnchorIDToDisk()");
+
+        string filename = "SavedAzureAnchorID.txt";
+        string path = Application.persistentDataPath;
+
+#if WINDOWS_UWP
+        StorageFolder storageFolder = ApplicationData.Current.LocalFolder;
+        path = storageFolder.Path.Replace('\\', '/') + "/";
+#endif
+
+        string filePath = Path.Combine(path, filename);
+        File.WriteAllText(filePath, currentAzureAnchorID);
+
+        Debug.Log($"Current Azure anchor ID '{currentAzureAnchorID}' successfully saved to path '{filePath}'");
+    }
+  
+    public void GetAzureAnchorIdFromDisk()
+    {
+        Debug.Log("\nAnchorModuleScript.LoadAzureAnchorIDFromDisk()");
+
+        string filename = "SavedAzureAnchorID.txt";
+        string path = Application.persistentDataPath;
+
+#if WINDOWS_UWP
+        StorageFolder storageFolder = ApplicationData.Current.LocalFolder;
+        path = storageFolder.Path.Replace('\\', '/') + "/";
+#endif
+
+        string filePath = Path.Combine(path, filename);
+        currentAzureAnchorID = File.ReadAllText(filePath);
+
+        Debug.Log($"Current Azure anchor ID successfully updated with saved Azure anchor ID '{currentAzureAnchorID}' from path '{path}'");
+    }
+
+    public void ShareAzureAnchorIdToNetwork()
+    {
+        Debug.Log("\nAnchorModuleScript.ShareAzureAnchorID()");
+
+        string filename = "SharedAzureAnchorID." + publicSharingPin;
+        string path = Application.persistentDataPath;
+
+#if WINDOWS_UWP
+        StorageFolder storageFolder = ApplicationData.Current.LocalFolder;
+        path = storageFolder.Path + "/";           
+#endif
+
+        string filePath = Path.Combine(path, filename);
+        File.WriteAllText(filePath, currentAzureAnchorID);
+
+        Debug.Log($"Current Azure anchor ID '{currentAzureAnchorID}' successfully saved to path '{filePath}'");
+
+        try
+        {
+            var client = new RestClient("http://167.99.111.15:8090");
```

Comment:
What's this hardcoded IP address about?  Seems like this should be a serialized property.  

Actually...I don't think `ShareAzureAnchorIdToNetwork` ever called.  Can you verify (search MRTK3 scripts, prefabs, and scenes).  If not used remove this function please

Reply:
Removed this and GetAzureAnchorIdFromNetwork since they seem not to be used